python/kserve/kserve/inference_client.py

```diff
@@ -499,7 +499,7 @@ async def infer(
         if isinstance(data, dict):
             data = orjson.dumps(data)
         response = await self._client.post(
-            url, content=data, headers=headers, timeout=timeout
+            url, content=data, headers=headers, timeout=120
```

Comment:
Is th is change expected?

Reply:
let me change this to a draft PR since I was just messing around with the timeouts